### DISCUSSION TITLE
Format for adding the license key to .env

### DISCUSSION THREAD

**Discussion by andylarkum** [lucky-media/cloud#14] [Q&A] [answered]
Hi

Purchased the license for Cloud starter-kit, and attempted to install into an existing statamic Sail. This fails, saying it requires the license key. The documentation suggests that the license key should be added to the .env file, but doesn't explain the format required!

So, please can you provide an example format for adding the license key to the .env file?

Thanks

Andy

**Comment by lokmanm** [comment]
Hi @andylarkum,

In order to enter the license it is required to do so when you first install the starter kit. Afterwards the license your website is requesting is for Statamic itself if you are using any of the pro features.

As for the Cloud license its only requested when you first install it and thats it. 

Let me know if I can help you somehow,

Thanks

**Comment by andylarkum** [comment, marked as answer]
Hi @lockmanm

Thanks for the quick reply. 
I'm sure I'm being thick here, so my apologies.

Here's what I'm doing:
```
sail php please starter-kit:install lucky-media/cloud
Validating starter kit license...
License required for [lucky-media/cloud]!
This is a paid starter kit. If you haven't already, you may purchase a license at:
https://statamic.com/starter-kits/luckymedia/cloud
```

I don't have an opportunity to enter the license to be validated, so I don't understand?

Thanks

A.

**Reply by andylarkum** [reply]
OK nevermind.
I installed a new site using Statamic CLI, and:
`composer require laravel/sail --dev`
...which achieved the same intended result

A.

**Reply by lokmanm** [reply]
Sorry a bit for the late reply. I was going to say that I havent tried installing it with Sail. Happy you found a solution, let me know if you need anything else. 

Thanks 👍
